satellite/gracefulexit/endpoint.go

```diff
@@ -187,6 +187,10 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 	ctx := stream.Context()
 	defer mon.Task()(&ctx)(&err)
 
+	if !endpoint.config.Enabled {
+		return nil
```

Comment:
Should we return an rpc error here to let the client know this is not enabled?

Reply:
Implemented the flag a different way. Now an "unimplemented" rpc error is returned